test/e2e/upgradetest/upgrade_test.go

```diff
@@ -38,8 +39,12 @@ func TestResize(t *testing.T) {
 			t.Fatal(err)
 		}
 	}()
-	if err = k8sutil.WaitEtcdTPRReady(testF.KubeCli.CoreV1().RESTClient(), 3*time.Second, 30*time.Second, testF.KubeNS); err != nil {
-		t.Fatalf("failed to see cluster TPR get created in time: %v", err)
+
+	lo := metav1.ListOptions{
```

Comment:
Is it better to abstract this to a comprehensive method?
Like WaitOperatorReady() ?